polaris-react/src/styles/shared/_page.scss

```diff
@@ -4,7 +4,7 @@
   max-width: $page-max-width;
 
   @include page-content-when-not-fully-condensed {
-    padding: 0 var(--p-space-6);
+    padding: 0 var(--p-space-6) var(--p-space-6);
```

Comment:
This could affect pages with `PageActions` at the bottom of the page. This appears to be a simple fix but will probably cause adverse white space on other pages.